src/app/shared/store/shared-user.actions.ts

```
@@ -70,3 +80,8 @@ export class ResetProviderWorkshopDetails {
   static readonly type = '[user] clear Provider And Workshop Details';
   constructor() {}
 }
+
+export class ResetProviderCompetitionDetails {
+  static readonly type = '[user] clear Provider And Competition Details';
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)